README.md

```diff
@@ -92,6 +92,28 @@ There are many other options to explore. You can see them all by running:
     pdoc --help
 
 
+Submodule loading
+-----------------
+
+`pdoc` uses idiomatic Python when loading your modules. Therefore, for `pdoc`
+to find any submodules of the input module you specify on the command line,
+those modules must be available through Python's ordinary module loading process.
+
+This is not a problem for globally installed modules like `sys`, but can be
+a problem for your own sub-modules depending on how you have installed them.
```

Comment:
Sure. To me that sounds like something to describe separately from the module loading issue, but I will add a quick note about that too. Hang on.